src/frontend/src/components/AlertAccordion/index.tsx

```diff
@@ -1,45 +1,40 @@
-import { IconButton } from "@material-ui/core";
-import { ExpandMore } from "@material-ui/icons";
-import { AlertTitle } from "@material-ui/lab";
-import { AlertProps } from "czifui";
+import { Collapse } from "@material-ui/core";
 import React, { useState } from "react";
-import { StyledAlert, Title } from "./style";
+import {
+  ColumnFlexContainer,
+  RowFlexContainer,
+  StaticSizeDiv,
+  StyledArrowDownIcon,
+  StyledArrowUpIcon,
+  StyledCallout,
+} from "./style";
 
 interface Props {
-  title?: string;
-  message: React.ReactNode;
-  severity: AlertProps["severity"];
-  className?: string;
+  collapseContent: React.ReactNode;
+  intent: "info" | "error" | "success" | "warning";
```

Comment:
why name this `intent` and not `severity`?

Reply:
that is sds convention